packages/clerk-js/src/ui/components/OrganizationProfile/BillingWidget.tsx

```diff
@@ -0,0 +1,44 @@
+import { useRouter } from '../../../ui/router';
+import { useOrganizationProfileContext } from '../../contexts';
+import { AlertIcon, Flex, Link, Text } from '../../customizables';
+
+export const BillingWidget = () => {
+  const { __unstable_manageBillingUrl, __unstable_manageBillingMembersLimit } = useOrganizationProfileContext();
+  const router = useRouter();
+
+  return (
+    <Flex sx={theme => ({ background: theme.colors.$blackAlpha50, padding: theme.space.$4 })}>
+      <AlertIcon
+        variant='warning'
+        colorScheme='danger'
+        sx={t => ({ marginTop: t.space.$1 })}
+      />
+      <Text
+        variant='regularRegular'
+        sx={t => ({
+          fontWeight: t.fontWeights.$normal,
+          lineHeight: t.lineHeights.$tall,
+        })}
+      >
+        The invitations could not be sent. Fix the following and try again: <br />
```

Comment:
@raptisj Didn't we change this copywriting? Please double-check with @alex-ntousias